### DISCUSSION TITLE
Please enable clipboard,left,right,select all option

### DISCUSSION THREAD

**Discussion by gsweeh** [florisboard/florisboard#1425] [Feedback]
Please enable clipboard,left,right,select all option work in all the time like in all webiste,apps in private mode or etc
As shown in photos
![IMG_20211205_190600](https://user-images.githubusercontent.com/65717972/144748916-2a3ad098-4909-4cda-bb4c-3882ee0b3600.jpg)
![IMG_20211205_190718](https://user-images.githubusercontent.com/65717972/144748919-b65c03d3-481a-4b3b-ac3f-593cb66fdabd.jpg)

**Comment by Aman9das** [comment]
Use the previous beta, it works fine.

**Reply by patrickgold** [reply]
That or wait for beta07, where it will be fully working again.
